.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x]
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
If we're dropping support for older node versions, this new requirement should also be updated on the "engines" section of the package.json file.

Reply:
IMO we should probably also bump the major package version.